docs/IntegrationTest.md

```diff
@@ -19,9 +19,21 @@ This depends on refactoring the Google Oauth Plugin.
 -->
 1. Make sure you have another cluster and service account set up for testing deployments as described in the GKE Plugin [usage documentation](Home.md#usage).
 
-1. Repeat steps in the [IAM Credentials](Home.md#iam-credentials) section on this new Jenkins instance with a new GCP service account.
-
-1. Create the following [RoleBinding](rbac/pipeline_it_bindings.yaml) which allows testing the namespace "test" when running `mvn verify`.
+1. Ensure a GCP Service Account has been created for testing. If needed, follow the instruction in the [Automation Option](Home.md#automation-option) section.
+
+1. Bootstrap the necessary cluster resources/roles for running integration tests:
+```bash
+export TEST_CLUSTER=<YOUR_TEST_CLUSTER_NAME>
```

Comment:
We should probably make these environment variables match up with the ones used in the ITs, or vice versa.
`GOOGLE_GKE_CLUSTER`
`GOOGLE_PROJECT_LOCATION`
`GOOGLE_PROJECT_ID`

Reply:
On second thought this isn't needed here because this may not be with the same machine for the integration tests, but I noticed we don't list those anywhere when they are needed to run the tests. For now I'll leave that out of this PR